src/cookbooks/bigquery/access.md

```diff
@@ -101,22 +113,29 @@ pip install google-cloud-bigquery
 python -c 'from google.cloud import bigquery; print([d.dataset_id for d in bigquery.Client().list_datasets()])'
 ```
 
-## From Spark
-We recommend the [Storage API Connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector) for accessing
-BigQuery tables in Spark as it is the most modern and actively developed connector. It works well with the BigQuery
-client library which is useful if you need to run arbitrary SQL queries (see example Databricks notebook) and load their
-results into Spark.
+### Spark
+
+#### Locally
+
+> This method requires [BigQuery Access](#BigQuery-Access-Request) to be provisioned.
+
+If you want to use Spark locally (or via an arbitrary GCP instance in the cloud), we recommend the [Storage API Connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector) for accessing BigQuery tables in Spark as it is the most modern and actively developed connector. It works well with the BigQuery client library which is useful if you need to run arbitrary SQL queries and load their results into Spark (see the [Databricks](#databricks) section for a link to an example notebook demonstrating this technique).
+
+#### Databricks
 
-### On Databricks
-The `shared_serverless_python3` cluster is configured with shared default GCP credentials that are automatically picked
-up by BigQuery client libraries. It also has the Storage API Connector library added as seen in the example
-[Python notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
+[Databricks Notebooks](https://docs.databricks.com/notebooks/index.html) provide an interactive
+computational environment, similar to Jupyter. If you are a Mozilla employee, you should be able to access it via [`sso.mozilla.com/databricks`](https://sso.mozilla.com/databricks).
 
-### On Dataproc
-Dataproc is Google's managed Spark cluster service. Accessing BigQuery from there will be faster than from Databricks
-because it does not involve cross-cloud data transfers.
+The `shared_serverless_python3` cluster is configured with shared default GCP credentials that are automatically picked up by BigQuery client libraries. It also has the Spark Storage API Connector library added as seen in the example [Python notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
```

Comment:
```suggestion
The `shared_serverless_python3` cluster is configured with shared default GCP credentials that are automatically received by BigQuery client libraries. It also includes the Spark Storage API Connector library that has been added as shown in the example [Python notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
```

Reply:
"that has been added" is not quite right here, actually added should not appear at all (my mistake). Will fix locally

Reply:
"received" isn't quite right, and also we shouldn't use "added". but 👍 on has->includes; will modify this before landing.